tools/rack/rack.plugin/META-INF/MANIFEST.MF

```diff
@@ -42,7 +42,8 @@ Require-Bundle: com.ge.research.jena,
  org.eclipse.ui.forms,
  org.slf4j.api,
  org.yaml.snakeyaml,
- org.eclipse.debug.ui
+ org.eclipse.debug.ui,
+ openjfx.swing;bundle-version="17.0.2"
```

Comment:
I'm surprised this requirement doesn't cause any error.  We are explicitly adding the javafx jars to this bundle's classpath instead of requiring Eclipse to find the javafx bundles in its P2 repositories.  At best, this requirement is unnecessary; at worst, it may be muddling where the swing classes come from (are they coming from a bundle in P2 or a jar in the classpath?).  We should remove this requirement since RITE still should work without it.

Reply:
Reverted the change. I suspect Eclipse inserted it.